api/v1beta1/awscluster_webhook_test.go

```diff
@@ -87,6 +87,20 @@ func TestAWSCluster_ValidateCreate(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "HealthCheckProtocol in ControlPlaneLoadBalancer should default to ClassicELBProtocolSSL",
+			cluster: &AWSCluster{
+				Spec: AWSClusterSpec{},
+			},
+			expect: func(t *testing.T, res *AWSLoadBalancerSpec) {
+				t.Helper()
+
```

Comment:
Can we use gomega `g := NewWithT(t)`
and also use same for comparing result with expected output

Reply:
Done